tests/unit_tests/subtensor/interface/test_type_registry.py

```diff
@@ -22,47 +22,46 @@
 import unittest
 import pytest
 
-from scalecodec.base import RuntimeConfiguration, ScaleType
+from scalecodec.base import RuntimeConfiguration
 
-from bittensor._substrate import SubstrateWSInterface, Keypair, SubstrateRequestException
+from substrateinterface import SubstrateInterface
 from .settings import *
 
 
 class KusamaTypeRegistryTestCase(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.substrate = SubstrateWSInterface(
+        cls.substrate = SubstrateInterface(
             address_type=2,
-            type_registry_preset='kusama'
+            type_registry_preset='kusama',
+            url='test.kusanagi.bittensor.com:99444'
```

Comment:
Potential Port typo

Reply:
seconded!

Reply:
Or.. perhaps it was intentional... 


...... no it wasn't. i'll take it out.